### DISCUSSION TITLE
Grove - LED Bar | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#135] [Q&A]
# Grove - LED Bar | Seeed Studio Wiki

Grove - LED Bar

https://wiki.seeedstudio.com/Grove-LED_Bar/

**Comment by skaundinyabci** [comment]
I received the LED Bar V2.0 QTY:2 today from Digikey. My goal is to  interface it with the Raspberry Pi. However, I was wondering if I can do that without needing to be buy the GrovePi+ Hat. I don't have any need for the all the extra sensor interfaces that the HAT can support. So where do I find information to directly control the LED bar without using the HAT?

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.
